typed_python/native_instance_wrapper.h

```diff
@@ -183,7 +183,7 @@ struct native_instance_wrapper {
 
     static PyObject* categoryToPyString(Type::TypeCategory cat);
 
-    static Instance tryUnwrapPyObjectToInstance(PyObject* inst);
+    static Instance unwrapPyObjectToInstance(PyObject* inst);
```

Comment:
drop the `try` from the name because we are setting the error if unable to unwrap (so it's not a graceful try method)